mobileApp/src/machines/gameMachine/idleState.ts

```diff
@@ -0,0 +1,19 @@
+import {MachineConfig} from 'xstate';
+import {GAME_TYPES, GameStates} from './stateTypes';
+import {makeAMove} from './helpers/makeAMove';
+import {IInitContext} from './gameMachine';
+import {GameEvents, GAMES_EVENTS} from './eventTypes';
+
+export const idleState = () => {
+    const idleStateConfig: MachineConfig<IInitContext, GameStates, GameEvents> = {
```

Comment:
Please, fix the type for all functions like this. idleStateConfig is not a Machine, it is a StateNodeConfig.